### PR TITLE
Switching templates logic style 

### PR DESCRIPTION
**Pull request by jbidoret**
Switching templates logic style to if:else:endif and foreach:endforeach. 
The first commit did no re-indentation, but for some files, I felt necessary to do some (which lead to quite complex diffs).

### CODE REVIEW

**Comment by vincent-peugnet** [comment]
Awesome !!!

mmh, I think I should rebase this branche on `master`, because the diff does not only show Julien's changes. I'll try something.

File: app/view/templates/homeopt.php [hunk 1]
```diff
@@ -22,12 +22,10 @@
...
                     <div>
                         <select name="sortby" id="sortby">
-                            <?php
-                            foreach (Wcms\Opt::SORTBYLIST as $col) {
-                                $name = Wcms\Model::METADATAS_NAMES[$col];
-                                echo '<option value="' . $col . '" ' . ($opt->sortby() == $col ? "selected" : "") . '>' . $name . '</option>';
-                            }
-                            ?>
+                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) :
+                                $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
```
Comment (vincent-peugnet) [inline app/view/templates/homeopt.php:26]:
```suggestion
                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) : ?>
                                <?php $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
```
or maybe this

Comment (vincent-peugnet) [reply app/view/templates/homeopt.php:26]: whatever, here I should have send better vars from the controller. like a good old assoc array ready to be used.

see #459

File: app/view/templates/homeopt.php [hunk 2]
```diff
@@ -130,8 +128,8 @@
                                         <span class="counter"><?= $count ?></span>
                                     </label>
                                 </li>
-                            <?php } ?>
-                        </ul>
+                            <?php endforeach ?>
+                        </ul>
```
Comment (vincent-peugnet) [inline app/view/templates/homeopt.php:132]:
```suggestion
                        </ul>
```

File: app/view/templates/user.php [hunk 3]
```diff
@@ -82,9 +80,7 @@
 
                     </tr>
 
-                    <?php
-                    }
-                ?>
+                    <?php endforeach ?>
```
Comment (vincent-peugnet) [inline app/view/templates/user.php:83]:
```suggestion
                <?php endforeach ?>
```

**Review by vincent-peugnet — CHANGES_REQUESTED** [review]

File: app/view/templates/home.php [hunk 4]
```diff
@@ -51,328 +51,332 @@
...
+                                        <td class="version"><?= $item->version() ?></td>                                
+                                    <?php endif ?>
+                                </tr>
+
+                            <?php endforeach ?>
+                        </tbody>
+                    </table>
+                </div>
+
+            <?php endif; // display ? ?>
```
[the remark this answers is not in the document]
Comment (jbidoret) [reply app/view/templates/home.php:379]: That was a “comment” (the if/else/endif block is big). Let’s remove it.

File: app/view/templates/home.php [hunk 5]
```diff
@@ -383,14 +387,14 @@ class="redirection"
...
+<?php if(!Wcms\Config::disablejavascript()) : ?>
 
 <script type="module" src="<?= Wcms\Model::jspath() ?>home.bundle.js"></script>
 
-<?php } ?>
+<?php endif ?>
 
 
-<?php } ?>
+<?php endif; // iseditor ? ?>
```
[the remark this answers is not in the document]
Comment (jbidoret) [reply app/view/templates/home.php:397]: The same :)

File: app/view/templates/home.php [hunk 6]
```diff
@@ -51,328 +51,332 @@
...
+                <?php endif ?>
                 <span class="right">
                     <a href="?display=list" <?= $display === 'list' ? 'class="selected"' : '' ?> ><i class="fa fa-th-list"></i></a>
                     <a href="?display=graph"  <?= $display === 'graph' ? 'class="selected"' : '' ?>  ><i class="fa fa-sitemap"></i></a>
                     <a href="?display=map" <?= $display === 'map' ? 'class="selected"' : '' ?> ><i class="fa fa-globe"></i></a>
                 </span>
             </h2>
 
-            <?php if($display === 'graph') { ?>
+        <?php if($display === 'graph') : ?>
```
[the remark this answers is not in the document]
Comment (jbidoret) [reply app/view/templates/home.php:73]: Yes

File: app/view/templates/homeopt.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (jbidoret) [reply app/view/templates/homeopt.php:26]: Don’t really know which way would be “cleaner”.

Comment (jbidoret) [reply app/view/templates/homeopt.php:26]: Meanwhile, this last one, more verbose but properly indented, seems to be better.

File: app/view/templates/delete.php [hunk 7]
```diff
@@ -16,13 +16,14 @@
...
             Page linking to this one : <?= $pageslinkingtocount ?>
-            <?php if ($pageslinkingtocount > 0) { ?>
+            <?php if ($pageslinkingtocount > 0) : ?>
                 <a href="<?= $this->url('home', [], '?linkto=' . $page->id() . '&submit=filter') ?>" title="search for pages linking to this one in home view">
                     <i class="fa fa-search"></i>
                 </a>
+            <?php endif ?>
         </li>
     </ul>
-    <?php } ?>
```
Comment (n-peugnet) [inline app/view/templates/delete.php:25]: Good catch!

File: app/view/templates/homeopt.php [hunk 8]
```diff
@@ -22,12 +22,10 @@
...
-                            <?php
-                            foreach (Wcms\Opt::SORTBYLIST as $col) {
-                                $name = Wcms\Model::METADATAS_NAMES[$col];
-                                echo '<option value="' . $col . '" ' . ($opt->sortby() == $col ? "selected" : "") . '>' . $name . '</option>';
-                            }
-                            ?>
+                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) :
+                                $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
+                                <option value="<?= $col ?>" <?= ($opt->sortby() == $col ? "selected" : "") ?> ><?= $name ?></option>
+                            <?php endforeach ?>
```
Comment (n-peugnet) [inline app/view/templates/homeopt.php:28]: Or simply (as $name is not used anywhere else):

```suggestion
                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) : ?>
                                <option value="<?= $col ?>" <?= ($opt->sortby() == $col ? "selected" : "") ?> >
                                    <?= Wcms\Model::METADATAS_NAMES[$col] ?>
                                </option>
                            <?php endforeach ?>
```

**Review by n-peugnet — APPROVED** [review]
Very good!

**Pull request merged**